src/examples/tags/TagsWeight.tsx

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import { Span } from '@zendeskgarden/react-typography';
+import { Tag } from '@zendeskgarden/react-tags';
+import { Row, Col } from '@zendeskgarden/react-grid';
+
+const Example = () => (
+  <Row>
+    <Col textAlign="center">
+      <Tag isRegular>
+        <Span>
+          <Span isBold>Category </Span>
+          Algae
+        </Span>
```

Comment:
Please replace with the following:

```
<span>
  <Span isBold>Category</Span> Algea
</span>
```

- no need for Garden `Span` unless applying a particular text font/color/weight treatment
- keep bolded text tight within surrounding tags (no trailing space)